PerformanceTests/JetStream2/JetStreamDriver.js

```diff
@@ -494,7 +496,7 @@ class Driver {
         if (!isInBrowser)
             return;
 
-        if (window.location.search !== '?report=true')
+        if (!urlParameters.get('report'))
```

Comment:
nit: 
```suggestion
        if (!urlParameters.has('report'))
```

Reply:
Hmm, this will not work as expected if I have `?report=false`.